New Lumifile (_v1), disabled metfilters in CommonModules

reimersa opened the pull request:
- Added lumifile based on `/afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions17/13TeV/ReReco/Cert_294927-306462_13TeV_EOY2017ReReco_Collisions17_JSON_v1.txt` announced here: https://hypernews.cern.ch/HyperNews/CMS/get/physics-validation/3067.html 

Lumi, calculated with `brilcalc lumi --normtag /cvmfs/cms-bril.cern.ch/cms-lumi-pog/Normtags/normtag_PHYSICS.json -u /fb -i /afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions17/13TeV/ReReco/Cert_294927-306462_13TeV_EOY2017ReReco_Collisions17_JSON_v1.txt`, is: **41.529 fb-1**

- Disabled MET filters in CommonModules as long as we don't store the flags

raggleton commented:
I thought the MET filters in data (and some newer MC) were OK? Seems a bit dangerous just disabling it completely. TBH I'd rather people use `CommonModules::disable_metfilters()` in their analysis code for specific datasets.

reimersa commented:
- Re-enabled MET filters in CommonModules
- Added JER files for 2017 JER
- Fixed typo in ElectronIDs that would cause a crash if trying to use Fall17 IDs
